test/inductor/test_flex_decoding.py

```diff
@@ -1545,6 +1557,18 @@ def test_mixed_dtypes_fails(self, device):
         ):
             flex_attention(query, key, value, _identity)
 
+    @supported_platform
```

Comment:
Can you please elaborate how those supported platforms are selected?

Reply:
For CPU, we support the flex attention template on platforms with AVX2, and also pursue the good performances on these platforms. Hence, we add the option of flex decoding here to do the optimization.
For the platforms without AVX2, we do not care the performance, so we recommend to use eager mode for them.